src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -96,7 +96,7 @@ internal static void UpdateBorderDrawable(this MaterialButton platformView, IBut
 				if (button.StrokeThickness > 0)
 					mauiDrawable.SetBorderWidth(button.StrokeThickness);
```

Comment:
Not part of the OG issue, but this looks scary similar...

Reply:
@mattleibow seems like ```StrokeThickness``` is behaving as expected in this case.